libr/core/cmd_open.c

```diff
@@ -328,9 +328,17 @@ static int cmd_open(void *data, const char *input) {
 	case '\0':
 	case '*':
 	case 'j':
-		r_core_file_list (core, (int)(*input));
+		if ('?' == input[1]) {
+			eprintf ("Usage: oj\n");
+		} else {
+			r_core_file_list (core, (int)(*input));
+		}
 		break;
 	case 'a':
+		if ('?' == input[1]) {
+			eprintf ("Usage: oa [addr]\n");
```

Comment:
Yeah, Im gonna fix it, but still block the execution when placed '?' in the end.
Actually wrote in a similar way of existed helps like [this](https://github.com/l4l/radare2/blob/oo-help/libr/core/cmd_open.c#L67)